common/common.go

```diff
@@ -751,23 +751,28 @@ func ListTriggersInFunc(c *cli.Context, client *fnclient.Fn, fn *modelsv2.Fn) ([
 func DockerRunInitImage(initImage string, fName string) error {
 	fmt.Println("Running init-image: " + initImage)
 
-	// Run the initImage
-	var c1ErrB bytes.Buffer
+	var errB bytes.Buffer
 	tarR, tarW := io.Pipe()
 
-	c1 := exec.Command("docker", "run", "--rm", "-e", "FN_FUNCTION_NAME="+fName, initImage)
-	c1.Stderr = &c1ErrB
-	c1.Stdout = tarW
-
-	c1Err := c1.Start()
-	if c1Err != nil {
-		fmt.Println(c1ErrB.String())
+	cmd := exec.Command("docker", "run", "--rm", "-e",
+		"FN_FUNCTION_NAME="+fName,
+		"-e", "http_proxy="+os.Getenv("http_proxy"),
```

Comment:
minor nit, it's probably better to not set these at all if env vars are not present. (as opposed to setting them to empty string)